### PR TITLE
First pass at HOSVD tutorial. Some outputs are inconsistent with MATL…

### PR DESCRIPTION
**Pull request by jeremy-myers**
…AB, so some work still remains to understand where the source of error is coming from.

----
:books: Documentation preview :books:: https://pyttb--247.org.readthedocs.build/en/247/

### CODE REVIEW

**Comment by ntjohnson1** [comment]
I'll let Danny review the PR but wanted to see if you had issues building the documentation locally. Following the contributing guide and running the sphinx build should also build the docs to locally validate tutorial changes/additions. Just figured your fresh set of eyes would be good to check that. It looks like your type hint was using python >=3.9 `tuple` where I believe readthedocs uses 3.8 `from typing import Tuple`. In which case the badge on the readme should update to show we support 3.8 once we release a new pypi package.

**Comment by jeremy-myers** [comment]
> I'll let Danny review the PR but wanted to see if you had issues building the documentation locally. Following the contributing guide and running the sphinx build should also build the docs to locally validate tutorial changes/additions. Just figured your fresh set of eyes would be good to check that. It looks like your type hint was using python >=3.9 `tuple` where I believe readthedocs uses 3.8 `from typing import Tuple`. In which case the badge on the readme should update to show we support 3.8 once we release a new pypi package.

I dropped the type hinting that caused the error. But I just experienced issues building the documentation locally. I'll open up an issue.

File: docs/source/tutorial/HOSVD.ipynb [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+    "## Simple example of usage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create random 50 x 40 x 30 tensor with 5 x 4 x 3 core\n",
```
Comment (dmdunla) [inline docs/source/tutorial/HOSVD.ipynb:53]: # Create random tensor of shape (50, 40, 30) with core of shape (5, 4, 3)\n"

NOTE: change all other occurrences in the text of "size" to "shape" and use tuples to show the specific shapes.

File: docs/source/tutorial/HOSVD.ipynb [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+    "# But a small multiple (i.e., |ndims(X)|) usually works...\n",
+    "print(\"\\nResult with tol = sqrt(3e-2):\")\n",
+    "T = ttb.hosvd(X, np.sqrt(3e-2), verbosity=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Similarly, lhe norm squared of the first block of G is 0.9, so specifying an error of `1e-1` *should* result in a core of size $3 \\times 2 \\times 1$"
```
Comment (dmdunla) [inline docs/source/tutorial/HOSVD.ipynb:285]: Similarly, the norm

File: docs/source/tutorial/HOSVD.ipynb [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"\\nTime for 10 runs of HOSVD:\")\n",
+    "%timeit -n10 T = ttb.hosvd(Y, tol=2 * np.sqrt(eps_machine), sequential=False, verbosity=0)"
+   ]
+  }
+ ],
+ "metadata": {
```
Comment (dmdunla) [inline docs/source/tutorial/HOSVD.ipynb:463]: Run the following before finalizing (from #208 which has not yet landed):

```
nbstripout docs/source/tutorial/class_ttensor.ipynb \
--extra-keys "metadata.language_info metadata.vscode metadata.kernelspec"
```

**Review by dmdunla — CHANGES_REQUESTED** [review]

**Comment by dmdunla** [comment]
When I run the blocks with `hosvd`, the output contains output as follows:

```
 Size of core: (7, 7, 7)
```

@jeremy-myers Can you add an Issue to change this to the following? (Remove the initial space and change "Size" to "Shape")

```
Shape of core: (7, 7, 7)
```

**Review by dmdunla — APPROVED** [review]

**Pull request merged**